server/belga/image.py

```diff
@@ -212,6 +212,93 @@ def format_list_item(self, data):
         }
 
 
+class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
+
+    GUID_PREFIX = 'urn:belga.be:360archive:'
+
+    label = 'Belga 360 Archive'
+    base_url = 'http://mules.staging.belga.be:48080/belga360-ws/'
+    search_endpoint = 'archivenewsobjects'
+    items_field = 'newsObjects'
+    count_field = 'nrNewsObjects'
+
+    def __init__(self, provider, **kwargs):
+        super().__init__(provider, **kwargs)
+        self.session = requests.Session()
+        self._id_token = None
```

Comment:
Is there any `auth` happening? If not, why do we have `self._id_token` and `self._auth_token`?